emsdk.py

```diff
@@ -142,9 +142,6 @@ def exit_with_error(msg):
   ARCH = 'x86'
 elif machine.startswith('aarch64') or machine.lower().startswith('arm64'):
   ARCH = 'aarch64'
-  if WINDOWS:
-      errlog('No support for Windows on Arm, fallback to x64')
-      ARCH = 'x86_64'
```

Comment:
You are right, as far as there are no publicly available Arm64 binearies we should instlal the x64 ones running in emulation. But having this code here prevents form possiblility to build the Arm64 binaries from sources using emsdk.py. For that reason I've move this check into the `install_tool()` method. The code is not nice but this should be only temporary solution.

Reply:
One option is that we drop the hacks and folks can use `EMSDK_ARCH=x86_64 emsdk install latest` if they want to force the install the x86 binaries?   Or do you think we should continue to make it automatic?

Reply:
I guess the hack is better because it keeps the current behaviour. I can rember when I started with emsdk that it took me a while to figure out that the `EMSDK_ARCH` even exists. On the other, hand note that there is this "strange" behavior: If you have Arm64 Python installed on your system and you do `emsdk.bat install latest` and `emsdk.bat activate latest`, the x64 version of emsdk Python is installed and activated and since then, `emsdk.bat install xxx` installs x64 versions anyway (without the error message). You either need to force Arm64 with `EMSDK_ARCH=aarch64` or install using `python emsdk.py install xxx`. If your system Python is x64, you need to force `EMSDK_ARCH=aarch64` in all cases.

Reply:
Yes, the python based system detection is a bit boring, as the architecture of your python dictates which platform you have. As you mentioned, if you have an x64 python (emulated on arm64), it will say you're on an x64 cpu.

The other way to detect this under Windows is to use PROCESSOR_ARCHITECTURE env var. Same joke, from cmd.exe, it says you're on an arm64 system, and from powershell, that you're on an x64 system (because default powershell install is x64).

For this kind of case, IMHO, having an explicit option (available through help) would be the best solution. Or at least, to mention ```EMSDK_ARCH``` in help and official documentation.